notebooks/20200721_TMPRSS2_SVM.ipynb

```diff
@@ -0,0 +1,544 @@
+{
```

Comment:
<p>I wouldn't save this file, best to only save the raw results. We'll need a new notebook which combines the results from the 3 methods, that notebook is the logical place to do this kind of processing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/srensi/tmprss2/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>